docs/source/release/v3.12.0/diffraction.rst

```diff
@@ -30,6 +30,7 @@ Powder Diffraction
 - After calling create_vanadium and focus in ISIS Powder scripts on POLARIS, the output workspaces always contain the sample material if it is set using set_sample_material. (To view the sample material, right click the workspace and click 'Sample Material...')
 - It is now possible to set beam parameters (height and width) using instrument_object.set_beam_parameters(height=123, width=456).
 - The ``mode`` parameter for POLARIS in ISIS Powder now behaves as described in the documentation - it persists through function calls and is case insensitive
+- New algorithm :ref:`algm-RefineSatellitePeaks` that can find and integrate a satellite peaks at fractional HKL given a collection of "seed" peaks supplied by the user.
```

Comment:
"find and integrate [all] satellite peaks"
"given as collection of fractional "seed" peaks"